cypress/e2e/persisting/normalizing.cy.ts

```diff
@@ -30,11 +30,11 @@ describe("The persist-button should create executable SQL", () => {
   });
 
   it("creates keys", () => {
-    cy.get('[data-cy="graph-element-columns"][class="ellipsis pk"]').should(
+    cy.get('[data-cy="graph-element-columns"].pk]').should(
```

Comment:
```suggestion
    cy.get('[data-cy="graph-element-columns"].pk').should(
```
then the test should run :)